### DISCUSSION TITLE
Snare midi notes

### DISCUSSION THREAD

**Discussion by ocimjr** [corrados/edrumulus#84] [General]
Hi, 
I was testing with a pd-80r, just one pad.
The only notes edrumulus produce for snare is a 38 center and 40 rimshot. Looking to replace my td-17, I see in Roland site that td-17 can produce a 37 as a cross stick.
https://support.roland.com/hc/en-us/articles/360005173411-TD-17-Default-Factory-MIDI-Note-Map

Is this something that will be implemented in the future or is already and I missing a config.

Thanks again

**Comment by corrados** [comment]
It is not yet supported but on the roadmap: https://github.com/corrados/edrumulus/blob/main/doc/TODO.md
